### PR TITLE
fix: Session table styling

### LINKED ISSUE #6221: Fix Session Table Columns for Small Resolutions and Larger Fonts
If a user has browser options that show the font as larger in many resolutions the table on the sessions page is distorted a bit. Also if the user has a small resolution this can happen. Please help to make the table work in more resolutions. In your changes improve the following:

* Make the status button on the left, e.g. "Confirmed" into one line
* Make "Lock Box" square and avoid that it touches the right side of the table
* Ensure Lock Session heading is in one line
* Ensure table borders/spacing between columns is same everywhere (it looks different on some columns)
* Ensure dates/times always fit into one row
* Increase width of speakers columns a bit to also show slightly longer speaker names in one row
* Get rid of the internal scroll bar on the bottom of the table in some resolutions and when fonts are larger
* Improve anything else that could distort the view

Screenshot from 2021-01-03 11-11-35

Screenshot from 2021-01-03 11-13-48

### PR DESCRIPTION
**Pull request by Creatoon**
Fixes #6221 

#### Short description of what this resolves:
This PR enchances the look of session table.

- [x] Make the status button on the left, e.g. "Confirmed" into one line
- [x] Make "Lock Box" square and avoid that it touches the right side of the table
- [x] Ensure Lock Session heading is in one line
- [x] Ensure table borders/spacing between columns is same everywhere (it looks different on some columns)
- [x] Ensure dates/times always fit into one row
- [x] Increase width of speakers columns a bit to also show slightly longer speaker names in one row
- [x] Improve anything else that could distort the view
- [x] Get rid of the internal scroll bar on the bottom of the table in some resolutions and when fonts are larger
 
The last task is covered in the PR #6207 

Screenshot with normal font size

Screenshot from 2021-01-04 05-06-59

Screenshot after 10% increment in font-size(see 110% in url bar)

Screenshot from 2021-01-04 05-06-56

Screenshot after 20% increment in font-size

Screenshot from 2021-01-04 05-06-53

Screenshot after 33% increment in font-size

Screenshot from 2021-01-04 05-06-46

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by maze-runnar — DISMISSED** [review]

File: app/styles/partials/buttons.scss [hunk 1]
```diff
@@ -8,3 +8,7 @@
   box-shadow: 0 0 0 1px transparent inset;
   color: rgba(0, 0, 0, .95);
 }
+
+.max-content-button {
```
Comment (iamareebjamal) [inline app/styles/partials/buttons.scss:12]:
```suggestion
.w-max-content {
```

**Review by sachinchauhan2889 — CHANGES_REQUESTED** [review]
center align lock icon
sesionn

File: app/templates/components/ui-table/cell/events/view/sessions/cell-buttons.hbs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<UiDropdown @class="ui floating dropdown button">
+<UiDropdown @class="ui floating dropdown button max-content-button">
```
Comment (iamareebjamal) [inline app/templates/components/ui-table/cell/events/view/sessions/cell-buttons.hbs:1]: ?

Comment (Creatoon) [reply app/templates/components/ui-table/cell/events/view/sessions/cell-buttons.hbs:1]: @iamareebjamal yes sir,  I am doing this,  as committed your suggested changes from my phone,   I am changing it asap.

**Comment by Creatoon** [comment]
> center align lock icon
I think this will not look good , because every cell in the table have a property of text-align: left,
if i will make only lock-session cell centred specifically , then it will not look good, please check this effect once from your side.

**Pull request merged**